templates/apps/sumo-aws-apps/common-org/common-org-configrecorder-role.yaml

```diff
@@ -0,0 +1,187 @@
+
+        - PolicyName: config-policy-config
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Sid: AllowConfigActions
+                Effect: Allow
+                Action:
+                  - config:DescribeConfigurationRecorderStatus
+                  - config:DescribeConfigurationRecorders
+                  - config:DeleteDeliveryChannel
+                  - config:PutConfigurationRecorder
+                  - config:DescribeDeliveryChannels
+                  - config:StopConfigurationRecorder
+                  - config:StartConfigurationRecorder
+                  - config:PutDeliveryChannel
+                  - config:DeleteConfigurationRecorder               
+                Resource: '*'
+
+        - PolicyName: resource-groups-policy
+          PolicyDocument:
+            Version: 2012-10-17
+            Statement:
+              - Sid: ResourceGroupsPolicy
+                Effect: Allow
+                Action:
+                - resource-groups:DeleteGroup
+                - resource-groups:CreateGroup
+                - resource-groups:GetGroup
+                - resource-groups:GetTags
+                - resource-groups:ListGroups
+                - resource-groups:Tag            
+                Resource:
+                  - !Sub 'arn:${AWS::Partition}:resource-groups:*:${AWS::AccountId}:group/*'
+
+  ConfigRecorderRole:
+    Metadata:
+      cfn_nag:
+        rules_to_suppress:
+          - id: W28
+            reason: Resource found with an explicit name, this disallows updates that require replacement of this resource
+    Type: AWS::IAM::Role
+    Properties:
+      RoleName: !Ref ConfigRecorderRoleName
+      AssumeRolePolicyDocument:
+        Version: 2012-10-17
+        Statement:
+          - Effect: Allow
+            Principal:
+              Service:
+                - config.amazonaws.com
+            Action:
+              - sts:AssumeRole
+      Path: /
+      ManagedPolicyArns:
+        - !Sub "arn:${AWS::Partition}:iam::aws:policy/service-role/AWS_ConfigRole"
+      Policies:
+        - PolicyName: root
+          PolicyDocument:
+            Version: '2012-10-17'
+            Statement:
+              - Effect: Allow
+                Action: s3:GetBucketAcl
+                Resource: !Join ['', ['arn:aws:s3:::', !Ref BucketName]]
+              - Effect: Allow
+                Action: s3:PutObject
+                Resource: !Join ['', ['arn:aws:s3:::', !Ref BucketName, /AWSLogs/,
+                                      !Ref 'AWS::AccountId', /*]]
+                Condition:
+                  StringEquals:
+                    s3:x-amz-acl: bucket-owner-full-control
+              - Effect: Allow
+                Action: 
+                  - config:PutConfigurationRecorder
+                  - config:PutDeliveryChannel
+                Resource: !Sub 'arn:${AWS::Partition}:config:${AWS::Region}:${AWS::AccountId}:*'        
```

Comment:
I mean shouldn't this be `!Sub 'arn:${AWS::Partition}:config:${AWS::Region}:${AWS::AccountId}:*'` changed to `!Sub 'arn:${AWS::Partition}:config:*:${AWS::AccountId}:*'` does it need permission for only single region ?